modules/ROOT/pages/clauses/delete.adoc

```diff
@@ -9,6 +9,7 @@ For removing properties and labels, see the xref::clauses/remove.adoc[REMOVE] cl
 
 It is not possible to delete nodes with relationships connected to them without also deleting the relationships. 
 This can be done by either explicitly deleting specific relationships, or by using the `DETACH DELETE` clause. 
+It is also possible, as of Neo4j 5.14, to explicitly add the keyword `NODETACH`, which has the same behavior as omitting it.
```

Comment:
nitpick: I do not think we should add the Cypher versions in the text, so that the text does not become littered with version numbers for when features began and ended. The docs should simply be an up to date representation of the language. If there is a need for users to refer to specific versions of the docs (say 5.13) then that is a feature that should be handled in a more generic way - until then we should accept that the use case is not met.

Reply:
Yeah, I will see what @JPryce-Aklundh says as we have been adding a bit more versioning in the docs :) but probably a better way to do it for sure!

Reply:
When a new feature is released we handle it with both labels and more formal "this feature was released in ..." notes. 
For this case, maybe instead of mentioning it in the introductory paragraph, we can add a subsection under the current "Delete single node header" called "NODETACH", and this can begin with the standard "This feature was introduced in Neo4j 5.14." note - such a section could also include an example of how to use NODETACH - which I think is currently missing.